agimus_controller/agimus_controller/warm_start_base.py

```diff
@@ -1,6 +1,6 @@
 from abc import ABC, abstractmethod
 import numpy as np
-
+from typing import Tuple
```

Comment:
I would change the order of the depencies import. 

from abc import ABC, abstractmethod
from typing import Tuple

import numpy as np

Because typing is a standard library. But it might depend on the convention

Reply:
makes sense, let's follow PEP 8